src/config.js

```diff
@@ -218,6 +218,13 @@ export const TILEGRID_EXTENT = [2420000, 1030000, 2900000, 1350000]
  */
 export const LV95_EXTENT = [572215.44, 5684416.96, 1277662.37, 6145307.4]
 
+/**
+ * TILEGRID_EXTENT (defined above) reprojected in EPSG:4326 through epsg.io website.
+ *
+ * @type {Number[]}
+ */
+export const WGS84_EXTENT = [5.1402988, 45.3981222, 11.4774363, 48.230617]
+
```

Comment:
maybe we should rename this two so that it makes a little bit more sense that they are related.
what about :
- `TILEGRID_EXTENT_EPSG_3857`
- `TILEGRID_EXTENT_EPSG_4326`

So that it is clear that they derive from `TILEGRID_EXTENT`

Reply:
done